PandasWorkshop.ipynb

```diff
@@ -21,7 +21,7 @@
   },
```

Comment:
<p>Could we make per capita here be per 1,000,000 people? That would make these numbers much nicer :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/orioncohen/metal-bands-by-nation/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>